administrator/components/com_banners/src/View/Banners/HtmlView.php

```diff
@@ -73,7 +73,7 @@ protected function initializeView()
         parent::initializeView();
 
         /** @var BannersModel $model */
-        $model               = $this->getModel();
+        $model            = $this->getModel();
```

Comment:
```suggestion
        $model = $this->getModel();
        $model->setUseExceptions(true);
```
It seems you've missed that when resolving the conflict?

Reply:
@richard67 banners is using ListView, we added the exception call to the ListView in the library, so is not needed here, AFAIU.  See https://github.com/joomla/joomla-cms/pull/45995/commits/f200a7deed8c5867902bdeb57417d6d1178b237f#diff-2d333764d3b745e8d516fc4bffc5967a4c28d23157e00b9464e239d28f1c0c69

Reply:
Ok, if that works. I've only noticed that in 5.4-dev that call was at that place. Maybe that has changed in 6.0-dev then with another PR.